src/http/server.clj

```diff
@@ -217,17 +217,25 @@
      :headers {"content-type" (str content-type "; charset=UTF-8")}
      :body body}))
 
+(defn matches-glob? [file glob]
+  (.matches
+   (.getPathMatcher
+    (java.nio.file.FileSystems/getDefault)
+    (str "glob:" "**/nasus/"  glob))
```

Comment:
The directory might have a different name 🤔

Reply:
oops - sorry about that!  Had my cloned dir in there while testing and forgot to change to pwd.